lib/mixlib/log.rb

```diff
@@ -30,7 +31,7 @@ module Log
 
 
     def reset!
-      @logger, @loggers = nil, nil
+      @logger, @loggers, @warned_from = nil, nil
```

Comment:
Probably means this to be `= nil, nil, nil`. Related, if you put these on their own lines these bugs would be more obvious :)